src/main/kotlin/no/nav/syfo/narmestelederrelasjon/database/NarmesteLederRelasjonQuery.kt

```diff
@@ -132,7 +132,6 @@ const val queryGetNarmesteLederRelasjonList =
     SELECT *
     FROM narmeste_leder_relasjon
     WHERE (arbeidstaker_personident = ? OR narmeste_leder_personident = ?)
-    AND aktiv_tom IS null;
```

Comment:
Vi må ha med de radene der aktiv_tom != null også siden det er de som gjør at en tidligere nærmeste leder blir satt til inaktiv.

Reply:
Dette skjønte jeg ikke 🤔 Nå får vi jo flere rader, men problemet som skal løses er at det kom for mange rader? 🤷

Reply:
Ja, for radene prosesseres etterpå, og da grupperes de og så velges den nyeste fra hver gruppe. Slik det var kommer alle tidligere nærmeste ledere ut som aktive.

Reply:
Ja, og nei. Den raden som finnes der endres ikke, men det kommer en ny rad med samme referanseId der tildatoen og riktig status er satt. Det er derfor vi må ta med rader der tildato != null når vi skal hente ut hele historikken. 

Jeg tror ikke vi har oppdaget det før siden vi oftest bare henter ut den første i lista, og det har blitt den nyeste (og den riktige).

Reply:
Skjønner, bra du fant det da 🏆